TestAssets/TestProjects/LightupLib/project.json

```diff
@@ -0,0 +1,386 @@
+{
+
+  "buildOptions": {
+    "emitEntryPoint": false
+  },
+  "dependencies": {},
+  "frameworks": {
+    "netcoreapp2.0": {
+      "dependencies": {
+        "Microsoft.NETCore.App": {
```

Comment:
Actually, thinking more about this.  I've spoken with @ericstj in the past about this type of scenario.  See [https://github.com/dotnet/core-setup/issues/635#issuecomment-284516681](https://github.com/dotnet/core-setup/issues/635#issuecomment-284516681) for a write-up on why using a `netstandard` library's .deps.json file at runtime is not correct.

Are the consumers of this feature aware that they should be producing these "Light up" deps.json files using netcore TFMs?

---
In reply to: [106215579](https://github.com/dotnet/core-setup/pull/1727#discussion_r106215579) [](ancestors = 106215579)